gulpfile.js/tasks/html.js

```diff
@@ -24,14 +24,17 @@ var htmlTask = function() {
     return JSON.parse(fs.readFileSync(dataPath, 'utf8'))
   }
 
+  var customFilters = TASK_CONFIG.html.customFilters || function(env) {}
+
   return gulp.src(paths.src)
     .pipe(data(getData))
     .on('error', handleErrors)
     .pipe(render({
       path: [path.resolve(process.env.PWD, PATH_CONFIG.src, PATH_CONFIG.html.src)],
       envOptions: {
         watch: false
-      }
+      },
+      manageEnv: customFilters
```

Comment:
let's expose the option as what it is. Rename `customFilters` to `manageEnv` and this looks good to go.